client/src/components/ModalDay/HeaderModalDay.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import styled from "styled-components";
+
+const Header = styled.header`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 0.5rem;
+  position: sticky;
+  top: 0;
+  background-color: #fff;
+  padding: 1rem;
+  input {
+    border: none;
+    border-bottom: 1px solid black;
+    margin-right: 1rem;
+    padding: 0.3rem;
+    font-family: "Orbitron", Arial, sans-serif;
+  }
+`;
+
+const HeaderModalDay = (
+  setModalDayOpen,
+  modalDayOpen,
+  onSubmit,
+  value,
+  onChange
+) => {
+  return (
+    <Header>
+      <button onClick={() => setModalDayOpen(!modalDayOpen)}>✖️</button>
+      <form onSubmit={onSubmit}>
```

Comment:
Maybe missing a `event.preventDefault()`

Reply:
It´s there, but it´s defined in the page file, where the onSubmit ist written. 😊